maps/pom.xml

```diff
@@ -3,14 +3,41 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.redhat</groupId>
-    <artifactId>kraken-parent</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
-  </parent>
-
+<!--  <parent>-->
```

Comment:
I presume the removal of the parent is for the s2i build?

Absolutely no way around that?

Reply:
Yes its for the S2I build. The only way around it would be to split the source trees into 2 different repos.

The parent pom is still there though and references maps & twitter as modules, so when using something like IntelliJ the UI still shows the 2 apps as modules, and you can still run `mvn` commands at the parent level.

Reply:
I assume this is because you want to rely on s2i's default location for picking up the generated artifacts - can't you tell s2i source build to take the artifacts from the proper submodule you want to include ?

Reply:
You can tell maven to build only a submodule - that's not the issue. The problem is the builder image is hard-core to look in `/target` for the final built artifact, which wont exist if still in a multi module build.

Reply:
Do we have an issue for s2i supporting this @maxandersen?

Reply:
@cescoffier isn't this similar to the .s2i config override to copy files in a separate step ?

Reply:
I raised https://github.com/quarkusio/quarkus-images/issues/60 for this.